uninstall.md

```diff
@@ -17,16 +17,24 @@ yum erase "*openshift*" "docker*"
 systemctl reset-failed
 systemctl daemon-reload
 
-# Remove data and configuration
+# Remove data, config, and training materials
 rm -rf /var/lib/openshift/
 rm /etc/sysconfig/*openshift*
 rm /etc/sysconfig/docker*
 rm -rf /etc/openshift/*
 rm -rf /root/.config/openshift
+rm -rf /root/training
+rm -rf /root/openshift-ansible
+rm -rf /etc/ansible
```

Comment:
I thought a default ansible installation is going to drop a few things in here.  Also, this would run on the ansible host, not on the hosts running openshift.